packages/std/src/mock.rs

```diff
@@ -158,9 +158,21 @@ impl MockQuerier {
 }
 
 impl Querier for MockQuerier {
-    fn query(&self, request: &QueryRequest) -> QuerierResult {
-        match request {
+    fn raw_query(&self, bin_request: &[u8]) -> QuerierResult {
+        // MockQuerier doesn't support Custom, so we ignore it completely here
```

Comment:
Why not?

Couldn't we pass a optional `Custom` handler or sub-queries like `self.bank` and `self.staking` into the `MockQuerier` constructor? This would support the idea of extending a querier instead of replacing it.